backend/src/email/routes/email-transactional.routes.ts

```diff
@@ -21,6 +21,45 @@ export const InitEmailTransactionalRoute = (
   const router = Router({ mergeParams: true })
 
   // Validators
+  const emailArrayValidation = (fieldName: 'cc' | 'bcc') => {
+    return Joi.alternatives().try(
+      // array
+      Joi.array()
+        .unique()
+        .items(
+          Joi.string().trim().email().options({ convert: true }).lowercase()
+        ),
+
+      // stringified array (form-data)
+      Joi.string().custom((value: string) => {
+        let parsed
+        try {
+          parsed = JSON.parse(value)
+        } catch {
+          throw new Error(
+            `${fieldName} must be a valid array or stringified array.`
+          )
+        }
+
+        if (!Array.isArray(parsed)) {
+          throw new Error(`${fieldName} must be a valid stringified array`)
+        }
+        const { value: validatedEmails, error } = Joi.array()
+          .unique()
+          .items(
+            Joi.string().email().trim().lowercase().options({ convert: true })
```

Comment:
`nit:` Could we either abstract or standardise the sequence of the standard validators for consistency?

It was `Joi.string().trim().email().options({ convert: true }).lowercase()` above.

Reply:
good point, have abstracted the email validation